packages/rollup-contracts/test/rollup-list/L1ToL2TransactionQueue.spec.ts

```diff
@@ -48,93 +46,39 @@ describe('L1ToL2TransactionQueue', () => {
     )
   })
 
-  const enqueueAndGenerateBlock = async (
-    block: string[],
-    blockIndex: number,
-    cumulativePrevElements: number
-  ): Promise<DefaultRollupBlock> => {
-    // Submit the rollup block on-chain
-    const enqueueTx = await l1ToL2TxQueue
-      .connect(l1ToL2TransactionPasser)
-      .enqueueBlock(block)
-    const txReceipt = await provider.getTransactionReceipt(enqueueTx.hash)
-    // Generate a local version of the rollup block
-    const ethBlockNumber = txReceipt.blockNumber
-    const localBlock = new DefaultRollupBlock(
-      ethBlockNumber,
-      blockIndex,
-      cumulativePrevElements,
-      block
-    )
-    await localBlock.generateTree()
-    return localBlock
-  }
-
-  /*
-   * Test enqueueBlock()
-   */
-  describe('enqueueBlock() ', async () => {
+  describe('enqueueBatch() ', async () => {
     it('should allow enqueue from l1ToL2TransactionPasser', async () => {
-      const block = ['0x1234']
-      await l1ToL2TxQueue.connect(l1ToL2TransactionPasser).enqueueBlock(block) // Did not throw... success!
+      await l1ToL2TxQueue.connect(l1ToL2TransactionPasser).enqueueTx(defaultTx) // Did not throw... success!
+      const batchesLength = await l1ToL2TxQueue.getBatchHeadersLength()
+      batchesLength.should.equal(1)
     })
     it('should not allow enqueue from other address', async () => {
-      const block = ['0x1234']
       await l1ToL2TxQueue
-        .enqueueBlock(block)
+        .enqueueTx(defaultTx)
```

Comment:
BTW there is a `TestUtils.assertRevertsAsync` which does this and auto-apends `VM Exception while processing transaction: revert ` :)